VRDR.Messaging/CodingResponseMessage.cs

```diff
@@ -140,14 +140,267 @@ public enum RaceCode
                 var list = new List<Tuple<string, Base>>();
                 foreach (KeyValuePair<RaceCode, string> item in value)
                 {
-                    var part = Tuple.Create(item.Key.ToString(), 
-                        (Base)(new Coding("https://www.cdc.gov/nchs/data/dvs/RaceCodeList.pdf", item.Value)));
+                    string codeList = "https://www.cdc.gov/nchs/data/dvs/RaceCodeList.pdf";
+                    // Special case for RACEBRG, it uses a different code document
+                    if(item.Key == RaceCode.RACEBRG) {
+                        codeList = "https://www.cdc.gov/nchs/data/dvs/Multiple_race_documentation_5-10-04.pdf";
+                    }
+                    var part = Tuple.Create(item.Key.ToString(),
+                        (Base)(new Coding(codeList, item.Value)));
                     list.Add(part);
                 }
                 Record.Add("race", list);
             }
         }
 
+        /// <summary>Coder Status (Numeric, 0-9)</summary>
+        public uint? CoderStatus
```

Comment:
I wonder if this should be a code instead of an integer? That way you could include a code system that references the list of statuses ?